spec/models/miq_report/generator_spec.rb

```diff
@@ -67,6 +67,23 @@
         expect(@miq_report_profile_all.table.data[0].data).to include("min_trend_value" => 400,
                                                                       "max_trend_value" => 700)
       end
+
+      it "handles a basic report with a where_clause and a where clause passed in options" do
```

Comment:
Maybe this, as a way to clarify this is dealing with "merging WHERE clauses":

```ruby
it "handles merging WHERE clauses from MiqReport#where_clause and options[:where_clause]"`
```

Reply:
Also, I assume this spec passed prior to the change in `app/models/miq_report/generator.rb`, correct?

Reply:
yes, I wrote that spec to ensure I didn't break anything.

Reply:
nice nick. it does not work before. I'll track this one down

Reply:
Looks like Keenan looked into this and fixed it, so resolving.